sdk/attestation/azure-security-attestation/tests/test_attestation.py

```diff
@@ -132,12 +132,13 @@
 )
 
 
-class AttestationTest(AzureTestCase):
```

Comment:
`TestAttestationTest` is redundant. 

I suggest removing one of the two "Test" elements, unless this is a test to verify that the `AttestationTest` class is correct (it doesn't appear to be - it appears to be a test for the Attestation APIs).

Reply:
Done